src/schemas/genes.json5

```diff
@@ -0,0 +1,24 @@
+{
+  $schema: 'http://json-schema.org/draft-07/schema',
+  description: 'Schema for the `genes` collection.',
+  version: 1,
+  type: 'object',
+  required: [],
+  properties: {
+    note: {type: 'string'},
+    codon_start: {type: 'string'},
+    inference: {type: 'string'},
+    location_end: {type: 'integer', minimum: 0},
+    location_start: {type: 'integer', minimum: 0},
```

Comment:
So gene locations can be actually pretty complex because genes can be spliced together from multiple locations on the genome. The "Start" and "End" supplied by BioPython can be pretty deceptive/useless for Euks and is also significantly different from how KBase stores locations.

It's not exactly how I would have designed it but locations in KBase are lists of triples of (contig, start_location, length)

Reply:
The location start and end fields are directly in the source data from NCBI, so I don't think it's supplied by biopython in any case I've seen; biopython just parses it out. All the data in this prototype schema is just a direct pull from the data points in NCBI. I'm not really doing any transformation. I'll look more at the KBase workspace type definitions